RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -763,3 +717,11 @@ def processDeepProducer(self, producer_name, tauIDSources, workingPoints_):
                         self.cms.InputTag(producer_name, 'VS{}{}'.format(target, point)))
 
             setattr(getattr(self.process, producer_name), 'VS{}WP'.format(target), cuts)
+
+
+    def getDpfTauVersion(self, file_name):
+        version_search = re.search('201[125678]v([0-9]+)[\._]', file_name)
+        if not version_search:
+            raise RuntimeError('File "{}" has an invalid name pattern. Unable to extract version number.'.format(file_name))
```

Comment:
Please add to description of the error information on an expected version label as in a comment above.